src/utils/JobLogUtils.py

```diff
@@ -5,30 +5,31 @@
 
 from config import config
 
-def GetJobLog(jobId, cursor=None, size=None):
+def GetJobLog(jobId, size=10000):
```

Comment:
if more than 10000 lines of logs are generated between two get job log requests, then we are missing logs?

Reply:
Resolved via elasticsearch scroll/scan, no more 10000 lines limit